### PR TITLE
Fix eslint ignitus-Testimonial

### LINKED ISSUE #54: Fix ESLINT Warnings
On running ```npm run lint``` ESLINT is showing 1081 problems (1075 errors, 6 warnings)
 621 errors and 0 warnings potentially fixable with the `--fix` option.
Fix as much as you can then feel free to send the PR.
We will be more than happy to merge it 👍 

screen shot 2018-10-07 at 11 05 29 am

### PR DESCRIPTION
**Pull request by darrylsepeda**
fix #54

Fix eslint ignitus-Testimonial

## Description
I splitted up my [previous pull request](https://github.com/Ignitus/Ignitus-Client-Side-Development/pull/57) to become smaller, hopefully can help the reviewer to easily merge this PR.

## Motivation and Context
Removing eslint errors from the existing source code

## How Has This Been Tested?
By running it locally

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.

### CODE REVIEW

File: src/ignitus-Api/constants.js [hunk 1]
```diff
@@ -1 +1,3 @@
-export const TESTIMONIAL_URL = 'https://ignitusrestapi.herokuapp.com/testimonials';
+const TESTIMONIAL_URL = 'https://ignitusrestapi.herokuapp.com/testimonials';
+
+export default TESTIMONIAL_URL;
```
Comment (outsourcegb) [inline src/ignitus-Api/constants.js]: @darrylsepeda , ignore export default. We may have more constants.

File: src/ignitus-Api/index.js [hunk 2]
```diff
@@ -1,8 +1,4 @@
-
-
 import axios from 'axios';
-import * as t from './constants';
+import TESTIMONIAL_URL from './constants';
```
Comment (outsourcegb) [inline src/ignitus-Api/index.js]: Check my above comment for the same. We may have more constants here.

File: src/ignitus-Testimonial/constants.js [hunk 3]
```diff
@@ -1 +1,3 @@
-export const API_URL = 'https://ignitusrestapi.herokuapp.com/testimonials';
+const API_URL = 'https://ignitusrestapi.herokuapp.com/testimonials';
```
Comment (outsourcegb) [inline src/ignitus-Testimonial/constants.js]: Can we create one variable like 
const API_URL = 'https://ignitusrestapi.herokuapp.com in api contatns , get it here and append?

File: src/ignitus-Api/constants.js (hunk 1 above)
Comment (divyanshu-rawat) [reply src/ignitus-Api/constants.js]: yes @outsourcegb 👍

File: src/ignitus-Api/index.js (hunk 2 above)
Comment (divyanshu-rawat) [reply src/ignitus-Api/index.js]: yes, please ignore default.

File: src/ignitus-Testimonial/constants.js (hunk 3 above)
Comment (divyanshu-rawat) [reply src/ignitus-Testimonial/constants.js]: yes it's fine @outsourcegb but we will be writing microservices and other restapi as well in the future? although creating approach is also fine but not exporting that one variable as default.

File: src/ignitus-Api/constants.js (hunk 1 above)
Comment (darrylsepeda) [reply src/ignitus-Api/constants.js]: noted, will be changed on the next patch

File: src/ignitus-Api/index.js (hunk 2 above)
Comment (darrylsepeda) [reply src/ignitus-Api/index.js]: okay

File: src/ignitus-Testimonial/constants.js (hunk 3 above)
Comment (darrylsepeda) [reply src/ignitus-Testimonial/constants.js]: FYI, this constant is not being used anywhere inside ignitus-Testimonial, plus I saw this API_URL is repeated among many components without being used (yet). My suggestion is we can put this constant.js at the root folder and use it anywhere inside the app.

So far these components exporting API_URL with the same value:

1. ignitus-Contributors/constant.js
2. ignitus-Introduction/constant.js
3. ignitus-Navigation/constant.js
4. ignitus-GetStarted/constant.js
5. ignitus-Testimonial/constant.js

**Comment by trblackw** [comment]
I agree

**Pull request closed without merging**